common/config/presets/d_console_wallet.toml

```diff
@@ -323,7 +323,7 @@ database_url = "data/wallet/dht.db"
 #network_discovery.enabled = true
 # A threshold for the minimum number of peers this node should ideally be aware of. If below this threshold a
 # more "aggressive" strategy is employed. Default: 50
```

Comment:
```suggestion
# more "aggressive" strategy is employed. Default: 8
```

Reply:
The default value in the struct is still set at 50; we change it here for the wallet.